### DISCUSSION TITLE
Can I specify a network interface to ban for the jail?

### DISCUSSION THREAD

**Discussion by SmirMax** [fail2ban/fail2ban#3616] [Q&A] [answered]
My server has multiple network interfaces, can I specify a network interface to ban on for the jail?

**Comment by sebres** [comment, marked as answer]
It depends on action, some of them need to be extended, another can get it from some parameter.
For instance for `iptables` based actions one can supply the interface within `chain` parameter.
```ini
[DEFAULT]
chain = INPUT -i eth0
[jail1]
chain = INPUT -i eth1
[jail2]
chain = INPUT
```
Or if expected to redefine action:
```ini
action = iptables-allports[chain="INPUT -i eth0"]
```
